chalice/build_deploy_config.sh

```diff
@@ -8,6 +8,12 @@ if [[ -z $DSS_DEPLOYMENT_STAGE ]]; then
 fi
 
 export stage=$DSS_DEPLOYMENT_STAGE
+export dss_infra_tag_project=${DSS_INFRA_TAG_PROJECT}
+export dss_infra_tag_service=${DSS_INFRA_TAG_SERVICE}
+export dss_infra_tag_stage=${DSS_DEPLOYMENT_STAGE}
+export dss_infra_tag_owner=${DSS_INFRA_TAG_OWNER}
+export dss_infra_tag_name=${DSS_INFRA_TAG_PROJECT}-${DSS_DEPLOYMENT_STAGE}-${DSS_INFRA_TAG_SERVICE}
```

Comment:
What's the purpose of renaming these variables?

Reply:
its to evaluate the env variable  in JQ without having to mess about with quote escaping, i realize that i had to perform it this way when i first started, i may be able to change this.